src/Button/GoogleLogin/stories.tsx

```diff
@@ -18,11 +18,7 @@
  */
 
 import { storiesOf } from '@storybook/react';
+import { GoogleLogin } from './index';
```

Comment:
couldn't pin down why, but some of the story files need the import with `./index` explicitly defined, others are fine as `.`